src/service/extension-location.js

```diff
@@ -59,32 +47,10 @@ export function calculateExtensionScriptUrl(location, extensionId, isLocalDev,
  * @param {!Location} location The window's location
  * @param {string} entryPoint
  * @param {boolean=} isLocalDev
- * @param {boolean=} isTest
  * @return {string}
  */
-export function calculateEntryPointScriptUrl(location, entryPoint, isLocalDev,
-    isTest) {
-  const base = calculateScriptBaseUrl(location, isLocalDev, isTest);
-  const serveMax = isLocalDev && isMax(location);
```

Comment:
Probably just an oversight (or deemed unnecessary). We'd also need `isUsingCompiledJs` to make sure we test min builds with `--compiled` here.